app/jobs/email_csv_job.rb

```diff
@@ -0,0 +1,16 @@
+class EmailCsvJob < ApplicationJob
+  queue_as :default
+
+  CSV_DOWNLOAD_TEMPLATE_ID = "QQ".freeze # Need an E-mail template + ID
```

Comment:
E-mail template required.

Reply:
@CheshireSwift Do you have access to Notify?

Reply:
@jamesrose Not that I'm aware of, assuming it's an independent login from other services?

Reply:
Yes it is. https://www.notifications.service.gov.uk/. You'll need to ask Gary Meyler to provide you a login (teams or email is best)